include/xtensor/xmath.hpp

```diff
@@ -25,6 +25,7 @@
 
 #include "xaccumulator.hpp"
 #include "xeval.hpp"
+#include "xmanipulation.hpp"
```

Comment:
This header should not be included here. Please rebase remove this and rebase your PR to fix the missing include.

Reply:
After the holidays. Sorry, it's family time until 7th of January

Reply:
Yeah no hurry, same here! Merry Christmas!

Reply:
Merry christmas, Hope i fixed it during the family IT service

Reply:
Ok it's not working, and that was exactly why i added the include

Reply:
My bad, I thought this was leading to potential circular includes (and I was trying to understand why the CI didn't catch it). Looks like circular dependencies occurs when `xmanipulation.hpp` is included in `xoperation.hpp`, not in `xmath.hpp`; sorry for the confusion. I'm revert the undesirable changes and drop the merge commits.

Reply:
No problem.